buổi 10/main.js

```diff
@@ -0,0 +1,98 @@
+// Bài 1:
+// Cách 1:
+const getArray = (string, lengthArray) => {
+  const arr = []
+  for(let i = 0; i < lengthArray; i++ ) {
+      arr.push(string)
+  }
+  return arr
+}
+
+//Cách 2:
+const getArray = (string, lengthArray) => {
+	let i = 0 
+	const arr = []
+	do {
+		arr.push(string)
+		i++
+	} while(i < lengthArray)
+	return arr
+}
+
+//Bài 2:
+const reArr = array => {
+  const newArr = []
+  for (let i = 0; i < array.length; i++) {
+    newArr.unshift(arr[i])
+  }
+  return newArr
+}
+
+//Bài 3:
+const deleteArray = (array = []) => {
+  const trueValues = []
+  for(let i = 0; i < array.length; i++) {
+    if(typeof(array[i]) == 'number' && array[i] !=0) {
+      trueValues.push(array[i])
+    }
+  }
+  return trueValues
+}
+
+//Bài 5:
+function sortArr(arr = []) {
+	arr.sort(function (a, b) { return a - b })
+}
+
+//Bài 6:
+const checkObject = obj => {
+	if (Array.isArray(obj) || typeof obj !== 'object') {
+		return false
+	} else {
+		return true
+	}
```

Comment:
Tab tào lao

Và nếu viết lệnh trên thì có thể viết ngắn gọn thành:
```suggestion
  return !Array.isArray(obj) && typeof obj === 'object'
```
Nó là obj nếu ko phải là array và có typeof là object


Nhưng vẫn sai (thiếu nên sai):
thử: `checkObject(null)` xem trả về cái gì